proto/internalapi/sensor/collector_iservice.proto

```diff
@@ -0,0 +1,18 @@
+syntax = "proto3";
+
+package sensor;
+
+import "google/protobuf/empty.proto";
+import "internalapi/common/collector_runtime_config.proto";
+
+option go_package = "sensor";
```

Comment:
I think this needs to be relative like `sensor_isservice.proto`.

`option go_package = "./internalapi/sensor;sensor";`

That was a change with the proto upgrade for the generation code to work correctly.  Should verify that with @janisz

Reply:
I have made this change in all of files in `common` as well.